Restructure auth docs to be more clear in function (#16303)
cockroachdb/docs: cockroachcloud/user-authorization.md

---
title: Managing User Authorization in CockroachDB Cloud

> summary: Learn procedures for managing the life cycle of CockroachDB Cloud users and roles.

Reviewer: nit (throughout): I prefer "lifecycle" -- maybe confirm which it should be?

> summary: Learn procedures for managing the life cycle of CockroachDB Cloud users and roles.

Author: ```suggestion
summary: Learn procedures for managing the lifecycle of CockroachDB Cloud users and roles.
```

toc: true
docs_area: manage
---

> {{ site.data.products.db }} has a two-level authorization model: users have permissions within organizations, and on specific database clusters within organizations.

Author: Yup, as noted there, this stuff is changing in the FGAC PR, so not trying to over-optimize here. I agree your line is clearer for what it's worth.

> Before proceding, it is recommended to review the {{ site.data.products.db }} authorization model, which is detailed in full here: [Authorization in {{ site.data.products.db }}](authorization.html)

Reviewer: ```suggestion
Before proceeding, it is recommended to review the {{ site.data.products.db }} authorization model, which is detailed in [Authorization in {{ site.data.products.db }}](authorization.html).
```

> This page documents procedures for managing the life cycle of CockroachDB Cloud users and roles, including:

Reviewer: ```suggestion
This page describes the procedures for managing organization users and roles:
```

> - Adding users to your organization, by creating them or inviting them from another organization

Reviewer: Initial caps and periods on list items.

- managing organization-level users roles
- granting organization users access to clusters through cluster roles
- managing the privileges associated with cluster roles

{{ site.data.products.db }} requires you to create SQL users to access the cluster.

> By default, a new SQL user created using a [Console Admin](console-access-management.html#console-admin) is assigned to the `admin` role. An `admin` SQL user has full privileges for all databases and tables in your cluster. This user can also create additional users and grant them appropriate privileges.

Reviewer: Should this info also be in `authorization.md`?

Author: nah, i don't think so, that's more about the model imo. in any case, this is changing with the upcoming release so it'll be moot.

> By default, a new SQL user created using a [Console Admin](console-access-management.html#console-admin) is assigned to the `admin` role. An `admin` SQL user has full privileges for all databases and tables in your cluster. This user can also create additional users and grant them appropriate privileges.

Reviewer: ```suggestion
By default, a new SQL user created using a [Console Admin](console-access-management.html#console-admin) is granted the `admin` role. An `admin` SQL user has full privileges for all databases and tables in the cluster, and can create additional SQL users and manage their privileges.
```

---
title: Managing User Authorization in CockroachDB Cloud

> summary: Learn procedures for managing the lifecycle of CockroachDB Cloud users and roles.

Reviewer: ```suggestion
summary: Learn how to manage the lifecycle of CockroachDB Cloud organization users and roles.
```

toc: true
docs_area: manage
---

{{ site.data.products.db }} has a two-level authorization model: users have permissions within organizations, and on specific database clusters within organizations.

Before proceeding, it is recommended to review the {{ site.data.products.db }} authorization model, which is detailed in [Authorization in {{ site.data.products.db }}](authorization.html).

> This page describes the procedures for managing organization users and roles:

Reviewer: ```suggestion
This page shows how to manage {{ site.data.products.db }} organization users and roles:
```

- Adding users to your organization, by creating them or inviting them from another organization.
- Managing organization-level users roles.
- Granting organization users access to clusters through cluster roles.
- Managing the privileges associated with cluster roles.

{{ site.data.products.db }} requires you to create SQL users to access the cluster.

> By default, a new SQL user created using a [Console Admin](console-access-management.html#console-admin) is granted the `admin` role. An `admin` SQL user has full privileges for all databases and tables in the cluster, and can create additional SQL users and manage their privileges.

Reviewer: ```suggestion
By default, a new SQL user created by a [Console Admin](console-access-management.html#console-admin) is granted the `admin` role. An `admin` SQL user has full privileges for all databases and tables in the cluster, and can create additional SQL users and manage their privileges.
```
